adcm_aio_client/core/objects/cm.py

```diff
@@ -480,15 +494,19 @@ def object(self: Self) -> Object:
     def action(self: Self) -> Action:
         return self._construct(what=Action, from_data=self._data["action"])
 
-    async def wait(self: Self, status_predicate: Callable[[], bool], timeout: int = 30, poll: int = 5) -> None:
-        if self._data["status"] not in (JobStatus.RUNNING, JobStatus.CREATED):
-            return
-
-        for _ in range(timeout // poll):
-            await asyncio.sleep(poll)
-            if status_predicate():
-                self._data["status"] = self.get_status()
-                return
+    async def wait(
+        self: Self,
+        timeout: int | None,
```

Comment:
we've decided the default timeout will be None

Reply:
`= 60` is not `None`

Reply:
changed